src/renderers/webgl/WebGLTextures.js

```diff
@@ -93,7 +93,7 @@ function WebGLTextures( _gl, extensions, state, properties, capabilities, utils,
 		_gl.generateMipmap( target );
 
 		var textureProperties = properties.get( texture );
-		textureProperties.__maxMipLevel = Math.log2( Math.max( width, height ) );
+		textureProperties.__maxMipLevel = Math.log( Math.max( width, height ) ) * Math.LOG2E;
```

Comment:
Personally I want comment here describing this style is for browsers which don't support Math.log2.

Reply:
Good point, I was intending to but forgot by the time I got to a keyboard.
Another + point for a polyfill, the workaround is contained in one place.

I'll add the comment